addons/website/static/src/snippets/s_searchbar/search_bar.js

```diff
@@ -216,6 +224,133 @@ export class SearchBar extends Interaction {
         }
     }
 
+    /**
+     * Handle keyboard navigation within search results
+     * @param {KeyboardEvent} ev
+     */
+    onSearchResultKeydown(ev) {
+        const allResults = [...this.menuEl.querySelectorAll(".o_search_result_item a")];
```

Comment:
```suggestion
        const allResults = this.menuEl.querySelectorAll(".o_search_result_item a");
```

Also, this will be called everytime you press the click, So ```numberOf(DOM Query) === NumberOf(key press)```.
To reduce this repetitive click, just cache this result in `this.allresults`  or somewhere else.

Reply:
This has to be done in render to atleast prevent querying the DOM for navigation keys. Its seems a fair tradeoff each keydown on input results in bringing new content and querying there seems more logical.